src/graphic/Fast3D/gfx_metal.cpp

```diff
@@ -199,6 +200,16 @@ bool Metal_IsSupported() {
 #endif
 }
 
+bool Metal_NonUniformThreadGroupSupported() {
+#ifdef __IOS__
+    // iOS devices with A11 or later support dispatch threads
+    return mctx.device->supportsFamily(MTL::GPUFamilyApple4);
```

Comment:
would this not return true on macs?

Reply:
Not sure, because this validation is for mobile cpus, and this worked as if was true always on macs, so i prefer to just returning true instead of doing a syscall